shared/react-native/android/app/src/main/java/io/keybase/ossifrage/KeyStore.java

```diff
@@ -62,91 +64,140 @@ private String keyStoreAlias(final String serviceName) {
     @SuppressLint("CommitPrefEdits")
     @Override
     public void clearSecret(final String serviceName, final String key) throws Exception {
-        prefs.edit().remove(sharedPrefKeyPrefix(serviceName) + key).commit();
+        String id = serviceName + ":" + key;
+        NativeLogger.info("KeyStore: clearing secret for " + id);
```

Comment:
vs the old code this is now logging a bunch of stuff, even in the happy path. i wonder if this'll hurt perf. we could just log the errors (the original goal)

Reply:
The secret store isn't in a hot path, and is consulted maybe a few times in a session, often only once at startup and if you log out/in. I don't think this'll hurt perf at all.

Reply:
i thought we used to his this a ton, i guess its cached now?

Reply:
service caches retrievesecret.

Reply:
can i see a logsend w/ this in it

Reply:
chat me a link